lib/cc/services/github_pull_requests.rb

```diff
@@ -22,6 +30,37 @@ class Config < CC::Service::Config
 
   private
 
+  def report_status?
+    if should_apply_rollout?
+      rollout_allowed_by_username? || rollout_allowed_by_percentage?
+    else
+      true
+    end
+  end
+
+  def should_apply_rollout?
+    (github_login.present? || github_user_id.present?) &&
+      (config.rollout_usernames.present? || config.rollout_percentage.present?)
```

Comment:
I think these has some spots in the boolean grid that wouldn't work (login present, but rollout-usernames not present). Should it be this:

```rb
(github_login.present? && config.rollout_usernames.present?) ||
  (github_user_id.present? && config.rollout_percentage.present?)
```

?